build-system/tasks/runtime-test.js

```diff
@@ -568,7 +568,13 @@ function runTests() {
             cyan('https://codecov.io/gh/ampproject/amphtml') + '...');
         const codecovCmd =
             './node_modules/.bin/codecov --file=test/coverage/lcov.info';
-        const output = getStdout(codecovCmd);
+        let flags = ' --flags=';
+        if (argv.unit) {
+          flags = flags + 'unit_tests';
+        } else if (argv.integration) {
+          flags = flags + 'integration_tests';
+        }
+        const output = getStdout(codecovCmd + flags);
```

Comment:
couldnt you end up with `--flags=` with empty value if unit or integration wasnt passed in?

Reply:
Good catch! It would've been a no-op since we never run coverage on Travis without `--unit` or `--integration`, but I've fixed it :)